### DISCUSSION TITLE
Discussion: Create project roadmap

### DISCUSSION THREAD

**Discussion by ahammednafih** [Eventy-devs/Eventy#25] [General]
XD link: https://xd.adobe.com/view/1ee6ac98-7a2c-45c2-ad57-ae01bf660ae9-4bb5/
Intro: https://vimeo.com/692727057/21cb4ff882 , https://youtu.be/dQoM7TCPzQA

**Comment by snehithlal** [comment]
@ahammednafih @Navinnair  Please check https://vimeo.com/692727057/21cb4ff882

**Comment by Navinnair** [comment]
@snehithlal thank you for the detailed explanation. Can we meet to discuss further on this? @ahammednafih @snehithlal
